app/api/admin/users/[id]/route.ts

```
@@ -0,0 +1,212 @@
+    }
+
+    const db = await getDatabase()
+    const user = await db.collection<User>('users').findOne({
+      _id: new ObjectId(id)
+    })
+
+    if (!user) {
+      return NextResponse.json({ error: 'User not found' }, { status: 404 })
+    }
+
+    const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)